apps/annotations/models.py

```diff
@@ -24,11 +29,13 @@ class Tag(TagBase, BaseTeamModel):
     name = models.CharField(verbose_name=pgettext_lazy("A tag name", "name"), max_length=100)
     created_by = models.ForeignKey("users.CustomUser", on_delete=models.DO_NOTHING, null=True, default=None)
     is_system_tag = models.BooleanField(default=False)
+    category_choices = [(category.value, category.name) for category in TagCategories]
```

Comment:
Nit: If you use a TextChoices rather than an Enum, then you can do this:
```python
category = models.CharField(choices=TagCategories.choices)
```

See for instance the [VoiceResponseBehaviours](https://github.com/dimagi/open-chat-studio/blob/11c1ce89b8533a36247ccf28b6ed631c7e79627a/apps/experiments/models.py#L289) enum and [its usage](https://github.com/dimagi/open-chat-studio/blob/11c1ce89b8533a36247ccf28b6ed631c7e79627a/apps/experiments/models.py#L402).

Reply:
[65479a4](https://github.com/dimagi/open-chat-studio/pull/440/commits/65479a45c0908473c634ea401f9b1dd0f0e6d049)